components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1763,7 +1791,9 @@ public void deleteGroup(String groupId) throws NotFoundException, CharonExceptio
             throw new NotImplementedException(error);
         }
 
-        if (SCIMCommonUtils.isFilteringEnhancementsEnabled()) {
+        if (isFilterUsersAndGroupsOnlyFromPrimaryDomainEnabled()) {
+            attributeValue = prependPrimaryDomain(attributeValue);
```

Comment:
For groups domains should be prepended only for group display name right
Won't this impact any other attribute